### PR TITLE
Fix #1081: updated get arguments in fragments

### LINKED ISSUE #1081: Update get arguments in fragments
updated get arguments from presenter to parent fragment for consistency.

### PR DESCRIPTION
**Pull request by nikitamarysolomanpvt**
## Explanation
updated get arguments in fragments

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/profile/ResetPinDialogFragmentPresenter.kt [hunk 1]
```diff
@@ -30,11 +30,13 @@ class ResetPinDialogFragmentPresenter @Inject constructor(
...
-    val profileId = fragment.arguments?.getInt(KEY_RESET_PIN_PROFILE_ID)
-    val name = fragment.arguments?.getString(KEY_RESET_PIN_NAME)
-    checkNotNull(profileId) { "Profile Id must not be null" }
-    checkNotNull(name) { "Name must not be null" }
+  fun handleOnCreateDialog(
+    routeDialogInterface: ProfileRouteDialogInterface,
+    profileId: Int,
+    name: String
+  ): Dialog {
+    val profileId = profileId
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/profile/ResetPinDialogFragmentPresenter.kt]: This is not needed.

File: app/src/main/java/org/oppia/app/profile/ResetPinDialogFragmentPresenter.kt [hunk 2]
```diff
@@ -30,11 +30,13 @@ class ResetPinDialogFragmentPresenter @Inject constructor(
...
+    val name = name
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/profile/ResetPinDialogFragmentPresenter.kt]: This is not needed.

File: app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragmentPresenter.kt [hunk 3]
```diff
@@ -42,14 +40,14 @@ class TopicPracticeFragmentPresenter @Inject constructor(
...
+    internalProfileId: Int,
+    topicId: String
   ): View? {
     val viewModel = getTopicPracticeViewModel()
-    val internalProfileId = fragment.arguments?.getInt(PROFILE_ID_ARGUMENT_KEY, -1)!!
-    topicId = checkNotNull(fragment.arguments?.getString(TOPIC_ID_ARGUMENT_KEY)) {
-      "Expected topic ID to be included in arguments for TopicPracticeFragment."
-    }
-    viewModel.setTopicId(topicId)
+    val internalProfileId = internalProfileId
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragmentPresenter.kt]: This can be removed

**Review by rt4914 — APPROVED** [review]
LGTM. Nit changes suggested.

File: app/src/main/java/org/oppia/app/profile/ResetPinDialogFragmentPresenter.kt (hunk 1 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/java/org/oppia/app/profile/ResetPinDialogFragmentPresenter.kt]: Done.

File: app/src/main/java/org/oppia/app/profile/ResetPinDialogFragmentPresenter.kt (hunk 2 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/java/org/oppia/app/profile/ResetPinDialogFragmentPresenter.kt]: Removed.

File: app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragmentPresenter.kt (hunk 3 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragmentPresenter.kt]: Removed.

**Pull request merged**